app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,9 +1,19 @@
 module Users
   class OmniauthCallbacksController < Devise::OmniauthCallbacksController
-    def mit_oauth2
+    def saml
       @user = User.from_omniauth(request.env['omniauth.auth'])
       sign_in_and_redirect @user, event: :authentication
       flash[:notice] = "Welcome #{@user.email}!"
     end
+
+    def developer
```

Comment:
This could stand to have a comment (and also the docs should be very clear about ensuring `FAKE_AUTH_ENABLED` is false in production).

Reply:
Super handy, though. I see why it's here.

Reply:
Cool. I can doc that up. Readme for sure and a note here.